webpack.config.js

```diff
@@ -94,45 +95,4 @@ const baseConfig = {
     },
 }
 
-/**@type {import('webpack').Configuration}*/
-const webviewConfig = {
-    name: 'vueLoader',
-    entry: {
-        samInvokeVue: path.resolve(__dirname, 'src', 'lambda', 'vue', 'samInvokeVue.ts'),
-    },
-    output: {
-        path: path.resolve(__dirname, 'dist', 'compiledWebviews'),
-        filename: '[name].js',
-    },
-
-    // Enable sourcemaps for debugging webpack's output.
-    devtool: 'source-map',
-
-    resolve: {
-        // Add '.ts' and '.tsx' as resolvable extensions.
-        extensions: ['.ts', '.tsx', '.js', '.json'],
-        alias: {
-            vue$: require.resolve('vue/dist/vue.esm.js'),
-        },
-    },
-    module: {
-        rules: [
-            {
-                test: /\.tsx?$/,
-                loader: 'esbuild-loader',
-                options: {
-                    loader: 'tsx',
-                    target: 'es2018',
-                },
-                exclude: /node_modules/,
-            },
-            {
-                test: /\.vue$/,
-                loader: 'vue-loader',
-            },
```

Comment:
I think the only valuable part of this PR is right here. The rest of the job is using `esbuild-loader` and will be necessary.

If you're tripping on the name, just rename it to something like `compileWebviews`

Reply:
Maybe I'm missing something but the webviews are being compiled separately in this PR. Compilation is based off the entry-points, so the webviews would still be separate from everything else.

Reply:
Maybe it was fixed with the move from esbuild to webpack? I know that that had issues with the target type.

Reply:
Well let's just wait for a bit on this PR to figure out what exactly is happening with these webviews. Insiders test is broken anyway.

Reply:
Confirmed in a separate branch that this works following an `npm run compile` (I guess esbuild handles frontend + backend better than webpack did).